sdk/src/schema/policy.ts

```diff
@@ -0,0 +1,24 @@
+import type { MergeCoreCollection } from '../index.js';
+import type { DirectusPermission } from './permission.js';
+import type { DirectusRole } from './role.js';
+import type { DirectusUser } from './user.js';
+
+/**
+ * directus_policies type
+ */
+export type DirectusPolicy<Schema> = MergeCoreCollection<
+	Schema,
+	'directus_policies',
+	{
+		id: string; // uuid
+		name: string;
```

Comment:
Has now also an icon 🎨 
```suggestion
		icon: string | null;
```

Reply:
Nitpick, the icon is always string as the column is non nullable